pydrive2/auth.py

```diff
@@ -347,6 +348,11 @@ def _InitializeStoragesFromSettings(self):
                     "Please specify credentials file to read"
                 )
             result[backend] = Storage(credentials_file)
+        elif backend == "dictionary":
```

Comment:
tbh, memory was better :) more human friendly. Minor comment.

Reply:
@shcheklein I thought so too, but it sounds kinda ambiguous if you think about other in-memory locations like plain str or some cryptic databases or even sqlite. Also this is more in-sync with oauth2client's naming, so probably better to be consistent.